### ISSUE TITLE
Most header information seems to be going missing somewhere

### ISSUE THREAD

**Issue by jacklenox** [Roguelazer/muttdown#19]
Hello, I don't send a huge amount of email from Mutt so I'm not sure how long this has been going on, but I've recently realised that when I send emails, almost all the header information appears to get discarded. At the receiving end I can't see the "Subject", or the full "From" name (just the raw email address is shown there). Emails sent via Muttdown also don't appear to be including my PGP electronic signature. Interestingly, the email in my "Sent items" looks exactly as it should and includes the PGP signature.

If I take Muttdown out of the picture, everything seems to be working fine.

I'm running Neomutt 20180716.

**Comment by Roguelazer** [comment]
How are you actually sending mail? Are you using Muttdown's built-in SMTP support or are you shelling out to sendmail? If the latter, do you know which sendmail you're using?

**Comment by jacklenox** [comment]
Yep, I'm shelling it out to sendmail with a line like this in my `.muttrc`:

```
set sendmail = "muttdown -c /home/jacklenox/.muttdown.yaml"
```

This definitely used to work, so I'm unsure what has caused the header information to start falling by the wayside.

**Comment by Roguelazer** [comment]
I have verified this locally and am working on a fix.

**Comment by Roguelazer** [comment]
The fix for this will be in the forthcoming 0.3.4.

**Comment by jacklenox** [comment]
Many thanks @Roguelazer!
